man/systemd-analyze.xml

```diff
@@ -188,6 +193,17 @@
     state. Its format is subject to change without notice and should
     not be parsed by applications.</para>
 
+    <para><command>systemd-analyze unit-paths</command> outputs a list of all
+    directories from which unit files, <filename>.d</filename> overrides, and
+    <filename>.wants</filename>, <filename>.requires</filename> symlinks may be
+    loaded. Combine with <option>--user</option> to retrieve the list for the user
+    manager instance, and <option>--global</option> for the global configuration of
+    user manager instances. Note that this verb prints the list that is compiled into
+    <command>systemd-analyze</command> itself, and does comunicate with the running
```

Comment:
"does communicate" → "does not communicate"?

Reply:
Oops.